mkdocs.yml

```diff
@@ -104,6 +104,13 @@ extra:
   
 nav:
   - Home: README.md
+  - Cookbook:
+    - Gradle Architecture: gradle-architecture/architecture/README.md
+    - Gradle on CI:
```

Comment:
This is where I start missing https://squidfunk.github.io/mkdocs-material/plugins/projects/

Reply:
We are going to have too many projects at the end ;)